src/React.js

```diff
@@ -64,4 +64,6 @@ assign(React, {
   ),
 });
 
+React.__sekret_DOM_DO_NOT_USE_OR_YOU_WILL_BE_FIRED = ReactDOM;
```

Comment:
wait, I thought this was a joke? In German, Sekret means secretion, so not exactly the word you are looking for here. Use "Geheimnis" instead.